OpenSim/Common/IMUHelper.h

```diff
@@ -0,0 +1,70 @@
+/* -------------------------------------------------------------------------- *
+ *                          OpenSim:  IMUHelper.h                           *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2019 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#ifndef OPENSIM_IMU_HELPER_H_
+#define OPENSIM_IMU_HELPER_H_
+
+// For definition of Exceptions to be used
+#include "DataAdapter.h"
+#include "FileAdapter.h"
+#include "TimeSeriesTable.h"
+#include "Event.h"
+
+/** @file
+* This file defines Helper class for reading or writing data files from IMU Makers.
+*/
+#include <map>
+
+namespace OpenSim {
+
+/** IMUHelper is a utility class that reads and writes files produced by IMU manufacturers
+    and produces datatables from them. This is intended to help consume IMU outputs,
+    files but no plan to create manufacturer specific formatted files.*/
+class OSIMCOMMON_API IMUHelper {
+public:
+    IMUHelper()                              = default;
+    IMUHelper(const IMUHelper&)            = default;
+    IMUHelper(IMUHelper&&)                 = default;
+    IMUHelper& operator=(const IMUHelper&) = default;
+    IMUHelper& operator=(IMUHelper&&)      = default;
+    virtual ~IMUHelper()                     = default;
+
+    static const std::string _orientations;   // name of table for orientation data
```

Comment:
This follows the convention in TRCFileAdapter, but if you prefer another convention let's make the changes to make all the code is consistent in this regard.

Reply:
No. `TRCFileAdapter` `static string _<string_name>` are private, yours are part of the public interface.

Reply:
Sorry, I meant C3DFileAdaptor here 
https://github.com/opensim-org/opensim-core/blob/3cc8808a7e84f7069eefb0d9ef533ca7331a6b0e/OpenSim/Common/C3DFileAdapter.h#L119
but regardless, I will change the capitalization rather than make it private since otherwise users have to guess the names of the tables.

Reply:
> 1. I think this class should be a proper `XSensDataReader` .